lib/zendesk_apps_support.rb

```diff
@@ -10,6 +10,7 @@ module ZendeskAppsSupport
   autoload :Package,                'zendesk_apps_support/package'
   autoload :Installed,              'zendesk_apps_support/installed'
   autoload :Installation,           'zendesk_apps_support/installation'
+  autoload :Finders,                'zendesk_apps_support/finders'
```

Comment:
not require a require call I assumed. since we aren't in rails and otherwise have to do the loading ourselves.